cmd/cstor-pool-mgmt/controller/replica-controller/handler.go

```diff
@@ -175,9 +175,16 @@ func (c *CStorVolumeReplicaController) cVRAddEventHandler(cVR *apis.CStorVolumeR
 		return string(apis.CVRStatusErrorDuplicate), nil
 	}
 
+	// Setting quorum to true for newly creating Volumes.
+	var quorum = true
+	if IsRecreateStatus(cVR) {
+		glog.Infof("Pool is recreated hence creating the volumes by setting off the quorum property")
+		quorum = false
+	}
+
 	// IsEmptyStatus is to check if initial status of cVR object is empty.
-	if IsEmptyStatus(cVR) || IsPendingStatus(cVR) {
-		err := volumereplica.CreateVolumeReplica(cVR, fullVolName)
+	if IsEmptyStatus(cVR) || IsPendingStatus(cVR) || IsRecreateStatus(cVR) {
```

Comment:
@mittachaitu Do we need handle the error status? If status is in `Error` state then CVR will never get created.
Is that Ok?

cc @vishnuitta